.travis.yml

```diff
@@ -18,6 +18,10 @@ script:
     - sudo cp shellcheck-${SCVERSION}/shellcheck /usr/bin/
     - shellcheck --version
     - make check
+    - pip install -U -r test-requirements.txt
+    - pip install PyGObject semver
```

Comment:
This stuff also starts to duplicate the requirements installed in the main RPM path (`deps.txt`).  I'm not opposed to this at all but it will be a maintenance burden.

Reply:
Agreed.